client/blocks/comments/post-comment.scss

```diff
@@ -31,6 +31,11 @@
 	.comments__form {
 		margin-top: 10px;
 	}
+
+	&:not(.depth-0) .comments__comment-actions .comments__comment-actions-reply-label,
+	&:not(.depth-0) .comments__comment-actions .reader-share__reblog-label {
+		display: none;
+	}
```

Comment:
do you think we should nest this in a breakpoint for smaller widths? Or just leave it this way everywhere?

Reply:
<img width="725" alt="Screenshot 2023-08-31 at 10 39 19 AM" src="https://github.com/Automattic/wp-calypso/assets/28742426/415bab64-cf93-414b-9cf2-5ccdfbc6c2d3">

they do feel fairly empty on the larger views